presto-docs/src/main/sphinx/release/release-325.rst

```diff
@@ -0,0 +1,44 @@
+===========
+Release 325
+===========
+
+General Changes
+---------------
+
+* Fix incorrect results for certain queries involving ``FULL`` or ``RIGHT`` joins and
+  ``LATERAL``. (:issue:`1952`)
+* Fix incorrect results when using ``IS DISTINCT FROM`` on columns of ``DECIMAL`` type
+  with precision larger than 18. (:issue:`1985`)
+* Fix query failure when row types contain a field named after a reserved SQL keyword. (:issue:`1963`)
+* Add support for ``LIKE`` predicate to ``SHOW SESSION`` and ``SHOW FUNCTIONS``. (:issue:`1688`, :issue:`1692`)
+* Add support for late materialization to join. (:issue:`1256`)
+* Reduce number of metadata queries during planning.
+  This change disables stats collection for non-EXPLAIN queries. If you
+  want to have access to such stats and cost in query completed event listener, you
+  need to re-enable stats collection using the ``collect-plan-statistics-for-all-queries``
+  property. (:issue:`1866`)
+* Add variant of :func:`strpos` that returns the Nth occurrence of a substring. (:issue:`1811`)
+* Add :func:`to_encoded_polyline` and :func:`from_encoded_polyline`. (:issue:`1827`)
+
+Web UI Changes
+--------------
+
+* Show actual query for an ``EXECUTE`` statement. (:issue:`1980`)
+
+Hive Changes
+------------
+
+* Fix incorrect behavior of ``CREATE TABLE`` when Hive metastore is configured
+  with ``metastore.create.as.acid`` set to ``true``. (:issue:`1958`)
+* Fix query failure when reading Parquet file with character data and no statistics. (:issue:`1955`)
+* Allow specifying column lists to analyze in ``ANALYZE``. (:issue:`1907`)
```

Comment:
How about

“Allow analyzing a subset of table columns (rather than all columns)”